dmoj/executors/COFFEE.py

```diff
@@ -7,7 +7,7 @@ class Executor(ScriptExecutor):
     ext = '.coffee'
     name = 'COFFEE'
     nproc = -1
-    fs = ['/etc/(?:resolv|nsswitch).conf$', '/$']
+    fs = ['/etc/(?:resolv|nsswitch).conf$', '/$', '/dev/pts/.*$']
```

Comment:
This is suboptimal, since it'd mean a judged process can read the commands of any user connected to a terminal. We do suggest people run the judge in its own user, which would raise EPERM when CoffeeScript tries accessing that. You can try replicating that like we used to in the SBCL executor:

https://github.com/DMOJ/judge/blob/a145b4f8a31a0e0c1232eb89458e8b8917256ac4/dmoj/executors/SBCL.py#L26-L43

(of course, `readlink` may not be the appropriate syscall here)